packages/integration-sdk-cli/src/neo4j/neo4jUtilities.ts

```diff
@@ -50,3 +55,25 @@ export function buildPropertyParameters(propList: Object) {
 
   return propertyParameters;
 }
+
+// Start and end type helper functions.  Prepends a : to any nonempty results for
+// immediate use in a Neo4j command.
+export function getFromType(relationship: Relationship): String {
+  if (relationship._fromType) {
+    return ':' + relationship._fromType.toString();
+  }
+  return '';
+}
+
+export function getToType(relationship: Relationship): String {
```

Comment:
The name `getToType` doesn't included that fact that a `:` will be prepended.
Maybe `getToTypeLabel`?